common/src/main/java/com/wynntils/features/user/overlays/ObjectivesOverlayFeature.java

```diff
@@ -181,8 +183,13 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
             final int barHeight = this.enableProgressBar ? 5 : 0;
             final int barWidth = 182;
             final float actualBarHeight = barHeight * (this.getWidth() / barWidth);
-            final float renderedHeight = SPACE_BETWEEN + 9 + actualBarHeight;
-            final float fullHeight = renderedHeight * objectives.size() - SPACE_BETWEEN;
+            final float renderedHeightWithoutTextHeight = SPACE_BETWEEN + actualBarHeight;
+            final float fullHeight = (float) (renderedHeightWithoutTextHeight * objectives.size()
```

Comment:
If we ever start doing something like this in some other place, I think we should move it to a utility class, but for now it can reside here.

Reply:
It's kind of icky. I assume it's doing the right thing but I'm a bit too tired to really wrap my head around it, and convince myself that it is. This is no complaint on your code, I believe the problem is actually more complex than one would think at first glance, and complex problems require complex solutions.